eng/Versions.props

```diff
@@ -32,7 +32,8 @@
     <MicrosoftCodeAnalysisTestingVersion>1.0.1-beta1.20623.3</MicrosoftCodeAnalysisTestingVersion>
     <CodeStyleAnalyzerVersion>3.8.0</CodeStyleAnalyzerVersion>
     <VisualStudioEditorPackagesVersion>16.8.181</VisualStudioEditorPackagesVersion>
-    <ILToolsPackageVersion>5.0.0-alpha1.19409.1</ILToolsPackageVersion>
+    <ILAsmPackageVersion>5.0.0-alpha1.19409.1</ILAsmPackageVersion>
+    <ILDAsmPackageVersion>5.0.0-preview.1.20112.8</ILDAsmPackageVersion>
```

Comment:
Why are we using different versions here?

Reply:
When I tried using the same, original version for both, NuGet complained me that it couldn’t find such a version of ildasm and chose the version that you see instead.